Dockerfile

```diff
@@ -32,6 +32,9 @@ ARG APP
 ENV WORKSPACE apps/${APP}
 # Create app directory
 WORKDIR /opt/openstad-headless
+
+RUN npm i -g nodemon
```

Comment:
Denk dat deze wijziging eigenlijk niet meer nodig is gezien deze al is opgenomen als devDependency bij de API?

Reply:
Goed gespot, is nu dubbelop inderdaad. Heb het nu weggehaald uit de Dockerfile!